src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -25,13 +25,15 @@ window.addEventListener(
   false
 );
 
-(function scrollToPosition() {
-  setTimeout(function () {
-    let scrollY = (document.body.scrollHeight - window.innerHeight) * ${position};
-    window.scrollTo(0, scrollY);
-    curPosition = scrollY;
-  }, 50);
-})();
+window.onload = function() {
```

Comment:
Can we use the arrow function syntax here to keep it consistent with other code.